snapcraft/internal/project_loader/_extensions/_utils.py

```diff
@@ -130,20 +131,51 @@ def _load_extension(
     )
 
 
+def _convert_to_ordereddict(property_list: List[Dict[str, str]]) -> OrderedDict:
+    od = collections.OrderedDict()
```

Comment:
should be type annotated with `od: OrderedDict = collections.OrderedDict()`

Reply:
also you'll have to rename `od` because the spell checker will complain... :(

Reply:
thank you so much!!! I didn't see the spell checker error (and still don't) but i renamed it to `myod`- hope this is ok.